identity/service/service.go

```diff
@@ -4,31 +4,28 @@ import (
 	"context"
 	"fmt"
 
-	"google.golang.org/grpc"
-
-	"github.com/plgd-dev/cloud/authorization/pb"
-	"github.com/plgd-dev/cloud/authorization/persistence"
-	"github.com/plgd-dev/cloud/authorization/persistence/mongodb"
+	"github.com/plgd-dev/cloud/identity/pb"
+	"github.com/plgd-dev/cloud/identity/persistence"
+	"github.com/plgd-dev/cloud/identity/persistence/mongodb"
 	"github.com/plgd-dev/cloud/pkg/log"
-	kitNetGrpc "github.com/plgd-dev/cloud/pkg/net/grpc"
 	"github.com/plgd-dev/cloud/pkg/net/grpc/server"
-	"github.com/plgd-dev/cloud/pkg/security/jwt"
 	"github.com/plgd-dev/cloud/pkg/security/jwt/validator"
 	"github.com/plgd-dev/cloud/resource-aggregate/cqrs/eventbus/nats/client"
 	"github.com/plgd-dev/cloud/resource-aggregate/cqrs/eventbus/nats/publisher"
 	"github.com/plgd-dev/cloud/resource-aggregate/cqrs/utils"
+	"google.golang.org/grpc"
 )
 
-// Provider defines interface for authentification against auth service
+// Provider defines interface for authentication against auth service
```

Comment:
pls remove the comment - it doesn't below to `Persistance`

Reply:
Removed